CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,16 @@
+- Set Nodejs 8.12.0 as minimum version in packages.json
 - Save cbHost in configuration
 - Upgrade Dockerfile to use NodeJS 8
 - Add: use PM2 in Dockerfile
-- Upgrade mongoose dependency to 4.13.14
+- Upgrade mongoose dependency from ~4.1.15 to 4.13.14

```

Comment:
Note (check packages.json) that mongosee depenceny has not been updated in this PR. This line has been modified to include the "from" version to align with the style used by the next ones.

NTC (just informative)